README.md

```diff
@@ -289,7 +289,7 @@ $ amtool silence expire $(amtool silence query -q)
 
 Amtool allows a config file to specify some options for convenience. The default config file paths are `$HOME/.config/amtool/config.yml` or `/etc/amtool/config.yml`
```

Comment:
"config" -> "configuration"
Also could you check for "Amtool" occurrences and replace them with `amtool` where it makes sense?